cpp/daal/src/algorithms/svm/svm_train_thunder_impl.i

```diff
@@ -421,7 +421,7 @@ services::Status SVMTrainImpl<thunder, algorithmFPType, cpu>::updateGrad(algorit
     DAAL_ITTNOTIFY_SCOPED_TASK(updateGrad);
 
     SafeStatus safeStat;
-    const size_t blockSizeGrad = 128;
+    const size_t blockSizeGrad = 64;
```

Comment:
Can this constant be derived from hardware specs?

Reply:
unlikely